Makefile.stubdom

```diff
@@ -67,7 +67,7 @@ build/busybox/config.status: build/busybox/.patched
 	touch $@
 
 build/busybox/busybox: build/busybox/config.status
-	$(MAKE) -C build/busybox
+	$(MAKE) $(MAKE_PARALLEL) -C build/busybox
```

Comment:
```suggestion
	$(MAKE) $(MAKE_FLAGS) -C build/busybox
```
I would suggest this instead of being too specific. In case we need more flags in the the future.

Reply:
You may also at least initialize it in the top of the Makefile as `MAKE_FLAGS ?=`

Reply:
> I would suggest this instead of being too specific. In case we need more flags in the the future.

We could go that route, but I fear `MAKE_FLAGS` is too close to `MAKEFLAGS` to be a reasonable name.

Also I'm really not sure we want to use a more generic name without extending it to all `make`invocations: I have expressly not added it to every one of them, since it makes little sense in some places (and we can doubt that every target was tested with `-j`) - and, that said, I was not so comfortable adding it to this pulseaudio invocation that runs build+install in one go.

Reply:
In fact, it would be much better to be able to do like other packages: just run `make %{?_smp_mflags}`.  I just went this route because doing so breaks.  Will have a close look at that.